content-services/latest/develop/sdk.md

```diff
@@ -2929,7 +2929,7 @@ worth noting that hot reloading is only supported on the platform, and not in Al
 
 As an alternative to the HotSwapAgent you can also try out JRebel. It has more features but isn't free.
 
-The way to configure HotSwapAgent in case of using Java 8 or Java 11 is pretty different. By default, ACS 6.0 uses Java 8 and ACS 6.1+ uses Java 11.
+The way to configure HotSwapAgent in case of using Java 11 or Java 17 is pretty different. By default, ACS 6.1 uses Java 11 and ACS 7.3+ uses Java 17.
```

Comment:
```suggestion
The way to configure HotSwapAgent in case of using Java 11 or Java 17 is pretty different. By default, ACS 6.1+ uses Java 11 and ACS 7.3+ uses Java 17.
```